<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Income_title_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Income Form</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="income_amount_linedit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Income Amount</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="income_resource_linedit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Income Resource</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="income_date_linedit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Income Date (yyyy/mm/dd)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="income_type_linedit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Income Type</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="income_discription_linedit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>321</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Discription</string>
    </property>
   </widget>
   <widget class="QPushButton" name="income_submit_btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>560</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>404</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
